### PR TITLE
alacritty, explorer, banner: add Chinese translation

### PR DESCRIPTION
**Pull request by ChungZH**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.zh/common/alacritty.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# alacritty
+
+> 跨平台，GPU 加速的终端模拟器。
```
Comment (einverne) [inline pages.zh/common/alacritty.md:3]:
```suggestion
> 跨平台，GPU 加速的终端模拟器.
```

Should end with a period.

File: pages.zh/common/banner.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# banner
+
+> 将给定参数输出为大型 ASCII 文字。
```
Comment (einverne) [inline pages.zh/common/banner.md:3]:
```suggestion
> 将给定参数输出为大型 ASCII 文字.
```

same here.

File: pages.zh/windows/explorer.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# explorer
+
+> Windows 文件资源管理器。
```
Comment (einverne) [inline pages.zh/windows/explorer.md:3]:
```suggestion
> Windows 文件资源管理器.
```

here.

Comment (ChungZH) [reply pages.zh/windows/explorer.md:3]: > In documents that mix Japanese and Latin scripts, the full-width comma (U+FF0C ， FULLWIDTH COMMA) is used; this is the standard form of comma (t 逗號, s 逗号, p dòuhào) in China. 
> [Comma - Wikipedia](https://en.wikipedia.org/wiki/Comma)

I think I did the right thing. :thinking: Haha

Comment (ChungZH) [reply pages.zh/windows/explorer.md:3]: 哦原来你也是中国人哈哈哈 :joy: 

或许我们可以参考参考这个：

image

:smile_cat: 如果你依然坚持，我也会修改的哈哈哈

File: pages.zh/common/alacritty.md (hunk 1 above)
Comment (zdroid) [reply pages.zh/common/alacritty.md:3]: This is the Chinese way to write a period, but right, those should be same in the whole page.

**Comment by ChungZH** [comment]
OK I will fix it tomorrow

**Pull request merged**